common/types/mapper/proto/history_test.go

```diff
@@ -363,3 +365,22 @@ func TestHistoryGetTaskInfoResponse(t *testing.T) {
 		assert.Equal(t, item, ToHistoryGetFailoverInfoResponse(FromHistoryGetFailoverInfoResponse(item)))
 	}
 }
+
+func TestRatelimitUpdate(t *testing.T) {
```

Comment:
Any specific reason not to follow table based approach that is used in other tests in this file? They are much shorter and easier to add more cases

Reply:
it'd be a moderate bit larger after accounting for the data in the other files.
but that's not really relevant here, this type probably shouldn't ever gain other fields.

those tests are also simpler because they're _only_ round-trip checks, not "input X produces output Y".  bug-for-bug compatibility between mappers won't be caught, nor missed fields (which is why I spent the time to figure out a fuzzing pattern too).